ibis/backends/tests/test_array.py

```diff
@@ -558,13 +646,41 @@ def test_array_sort(backend, con):
     raises=GoogleBadRequest,
     reason="BigQuery doesn't support arrays with null elements",
 )
-def test_array_union(con):
-    t = ibis.memtable({"a": [[3, 2], [], []], "b": [[1, 3], [None], [5]]})
+@pytest.mark.parametrize(
+    ("a", "b", "expected_array"),
+    [
+        param(
+            [[3, 2], [], []],
+            [[1, 3], [None], [5]],
+            [{1, 2, 3}, {None}, {5}],
+            id="including-empty-array",
+            marks=[
+                pytest.mark.notyet(
+                    ["flink"],
+                    raises=Py4JJavaError,
+                    reason=(
+                        "SQL validation failed; Flink does not seem to support ARRAY[]"
```

Comment:
```suggestion
                        "SQL validation failed; Flink does not support ARRAY[]"
```
Nit: If it indeed doesn't support it, maybe can take the doubt out of the statement :)

Reply:
Thanks, done.